model/src/main/kotlin/OrtResult.kt

```diff
@@ -200,10 +200,8 @@ data class OrtResult(
         val scannerIssues = scanner?.results?.collectIssues().orEmpty()
         val advisorIssues = advisor?.results?.collectIssues().orEmpty()
 
-        val analyzerAndScannerIssues =
-            analyzerIssues.zipWithDefault(scannerIssues, emptySet()) { left, right -> left + right }
-
-        return analyzerAndScannerIssues.zipWithDefault(advisorIssues, emptySet()) { left, right -> left + right }
+        val analyzerAndScannerIssues = analyzerIssues.zipWithCollections(scannerIssues)
+        return analyzerAndScannerIssues.zipWithCollections(advisorIssues)
```

Comment:
How about ditching the newline in 202 instead of the one above the return?

Reply:
I'd rather keep the two consecutive `zipWithCollections()` grouped.